wrench-app/src/test/java/com/izettle/wrench/provider/WrenchProviderTest.kt

```diff
@@ -1,43 +1,71 @@
 package com.izettle.wrench.provider
 
-import android.test.ProviderTestCase2
-import android.test.mock.MockContentResolver
 import androidx.room.Room
+import androidx.test.core.app.ApplicationProvider
 import androidx.test.ext.junit.runners.AndroidJUnit4
 import com.izettle.wrench.BuildConfig
 import com.izettle.wrench.core.Bolt
 import com.izettle.wrench.core.Nut
 import com.izettle.wrench.core.WrenchProviderContract
 import com.izettle.wrench.database.WrenchDatabase
-import com.izettle.wrench.preferences.WrenchPreferences
+import com.izettle.wrench.di.sampleAppModule
+import org.junit.After
 import org.junit.Assert
 import org.junit.Before
 import org.junit.Test
 import org.junit.runner.RunWith
-import org.mockito.Mockito.mock
+import org.koin.android.ext.koin.androidContext
+import org.koin.core.context.GlobalContext
+import org.koin.core.context.loadKoinModules
+import org.koin.core.context.startKoin
+import org.koin.core.context.stopKoin
+import org.koin.dsl.koinApplication
+import org.koin.dsl.module
+import org.koin.test.KoinTest
+import org.koin.test.get
+import org.robolectric.Robolectric
+
+val roomTestModule = module {
+    single(override = true) {
+        // In-Memory database config
+        val lol = Room.inMemoryDatabaseBuilder(ApplicationProvider.getApplicationContext(), WrenchDatabase::class.java).allowMainThreadQueries().build()
+        lol
+    }
+
+    single(override = true) {
+        TestPackageManagerWrapper("TestApplication", "com.test.application") as IPackageManagerWrapper
+    }
+}
 
 @RunWith(AndroidJUnit4::class)
-class WrenchProviderTest : ProviderTestCase2<WrenchProvider>(WrenchProvider::class.java, BuildConfig.CONFIG_AUTHORITY) {
+class WrenchProviderTest : KoinTest {
+
+    private lateinit var wrenchProvider: WrenchProvider
 
-    private var contentResolver: MockContentResolver? = null
+    private var databass: WrenchDatabase? = null
```

Comment:
Googled databass. Was a bit disappointed. 
http://engelsanchez.net/databass/databass.gif

Reply:
Haha. was not expecting anyone to do that. The property has been removed however - it was only added for debugging purposes.